history/util.py

```diff
@@ -0,0 +1,22 @@
+import psycopg2
+
+import server.secrets as secrets
+
+HISTORY_TABLE_NAME = "newtrains_history"
+
+HISTORY_DB_SCHEMA = """
+CREATE TABLE IF NOT EXISTS newtrains_history (
+    id UUID DEFAULT uuid_generate_v4(),
```

Comment:
This better than the nextval that's currently there?

Reply:
Not at all. I'm a UUID fiend and I didn't check the existing schema, but I agree that `nextval` is simpler!